bigtable/filters/filters_test.go

```diff
@@ -0,0 +1,706 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package filters
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/uuid"
+	"os"
+	"strings"
+	"testing"
+
+	"cloud.google.com/go/bigtable"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestFilters(t *testing.T) {
```

Comment:
Is there any way to convert this to a [table driven test](https://github.com/golang/go/wiki/TableDrivenTests)? All of the filters have the same signature. So, you could have a test like:

```go
tests := []struct{
	name string
	filter func(io.Writer, string, string, string) error
	want string
}{
	{
		name: "filterComposingCondition",
		filter: filterComposingCondition,
		want: "...",
	},
	// ...
}
```

This also makes it easy to run the tests in parallel using `t.Run` (a subtest) and `t.Parallel`.